data/json/recipes/electronic/components.json

```
@@ -15,6 +15,122 @@
     "qualities": [ { "id": "SCREW", "level": 1 } ],
     "components": [ [ [ "e_scrap", 1 ] ], [ [ "scrap", 1 ] ], [ [ "cable", 1 ] ] ]
   },
+  {
+    "type": "recipe",
+    "result": "motor",
+    "category": "CC_ELECTRONIC",
+    "subcategory": "CSC_ELECTRONIC_PARTS",
+    "skill_used": "electronics",
+    "difficulty": 3,
+    "skills_required": [ "fabrication", 2 ],
+    "time": "120 m",
+    "reversible": true,
+    "decomp_learn": 3,
+    "book_learn": [ [ "textbook_fabrication", 3 ], [ "manual_electronics", 3 ], [ "mag_electronics", 3 ], [ "manual_mechanics", 3 ] ],
+    "using": [ [ "soldering_standard", 40 ], [ "steel_standard", 5 ] ],
+    "qualities": [
+      { "id": "HAMMER", "level": 1 },
+      { "id": "SAW_M_FINE", "level": 1 },
+      { "id": "SAW_M", "level": 1 },
+      { "id": "SCREW", "level": 1 }
+    ],
+    "components": [ [ [ "e_scrap", 5 ] ], [ [ "power_supply", 1 ] ], [ [ "amplifier", 2 ] ], [ [ "bearing", 10 ] ], [ [ "cable", 20 ] ] ]
+  },
+  {
+    "type": "recipe",
+    "result": "motor_large",
+    "category": "CC_ELECTRONIC",
+    "subcategory": "CSC_ELECTRONIC_PARTS",
+    "skill_used": "electronics",
+    "difficulty": 5,
+    "skills_required": [ [ "fabrication", 2 ], [ "mechanics", 4 ] ],
+    "time": "300 m",
+    "reversible": true,
+    "decomp_learn": 4,
+    "book_learn": [ [ "textbook_fabrication", 4 ], [ "manual_electronics", 4 ], [ "mag_electronics", 4 ], [ "manual_mechanics", 4 ] ],
+    "using": [ [ "soldering_standard", 50 ], [ "steel_standard", 15 ] ],
+    "qualities": [
+      { "id": "HAMMER", "level": 1 },
+      { "id": "SAW_M_FINE", "level": 1 },
+      { "id": "SAW_M", "level": 1 },
+      { "id": "SCREW", "level": 1 }
+    ],
+    "components": [
+      [ [ "e_scrap", 5 ] ],
+      [ [ "power_supply", 2 ] ],
+      [ [ "amplifier", 4 ] ],
+      [ [ "bearing", 40 ] ],
+      [ [ "cable", 30 ] ],
+      [ [ "motor_oil", 500 ] ]
+    ]
+  },
+  {
+    "type": "recipe",
+    "result": "motor_enhanced",
+    "category": "CC_ELECTRONIC",
+    "subcategory": "CSC_ELECTRONIC_PARTS",
+    "skill_used": "electronics",
+    "difficulty": 6,
+    "skills_required": [ [ "fabrication", 4 ], [ "mechanics", 5 ] ],
+    "time": "480 m",
+    "reversible": true,
+    "decomp_learn": 6,
+    "book_learn": [ [ "textbook_robots", 5 ], [ "recipe_lab_elec", 5 ] ],
+    "using": [ [ "soldering_standard", 75 ], [ "steel_standard", 20 ], [ "welding_standard", 10 ] ],
```

Maybe less steel and more superalloy?
```suggestion
    "using": [ [ "soldering_standard", 75 ], [ "welding_standard", 10 ] ],
```